src/layer/S1GRDAWSEULayer.ts

```diff
@@ -139,6 +145,7 @@ export class S1GRDAWSEULayer extends AbstractSentinelHubV3Layer {
     }
     payload.input.data[0].processing.backCoeff = this.backscatterCoeff;
     payload.input.data[0].processing.orthorectify = this.orthorectify;
+    payload.input.data[0].processing.demInstance = this.demInstanceType;
```

Comment:
I'm not sure if it makes sense to always add  `demInstance` to payload. It might be better to set it only when ortorectification is enabled. fftr

Reply:
Probably as demInstanceType requires orthorectify to be set true. I updated it, can you please again.